.travis.yml

```diff
@@ -136,7 +140,11 @@ jobs:
       os: linux
       dist: xenial
       language: python
-      python: 2.7.14
+      python:
```

Comment:
Remove the whole `- stage: test` section.  Travis does not support multiple pythons in a single section, so you need to creat a new one for each python. And it is already done, see code below.